Recipes/Recipes.API/Program.cs

```diff
@@ -13,36 +18,63 @@
     app.UseSwaggerUI();
 }
 
-app.MapGet("/recipes", () =>
+app.MapGet("/recipes", async (RecipesDatabaseContext dbContext) =>
+{
+    // Ingredients
+    var spaghetti = new IngredientEntity() { Name = "Spaghetti noodles" };
+    var beef = new IngredientEntity() { Name = "Ground beef" };
+
+    dbContext.Ingredients.Add(spaghetti);
+    await dbContext.SaveChangesAsync();
+
+    dbContext.Ingredients.Add(beef);
+    await dbContext.SaveChangesAsync();
+
+    // Recipe
+    var recipe = new RecipeEntity()
     {
-        var recipes = new Recipe[]
+        Title = "Mom's Spaghetti",
+        Description = "BlaBla",
+        AuthorId = 1,
+        CookingTime = new TimeSpan(0, 2, 0, 0),
+        UpdatedAt = DateTime.UtcNow,
+        Servings = 3,
+        Instructions = new List<string>
+        {
+            "Cook spaghetti noodles according to package instructions.",
+            "In a large skillet, brown ground beef with chopped onion and minced garlic.",
+            "Add tomato sauce, Italian seasoning, salt, and pepper to the skillet. Simmer for 10 minutes.",
+            "Serve the spaghetti topped with the meat sauce and sprinkle with Parmesan cheese."
+        }
+    };
+    spaghetti = dbContext.Ingredients.Find(1);
+    beef = dbContext.Ingredients.Find(2);
+    recipe.Ingredients.Add(new RecipeIngredient { Ingredient = spaghetti, Quantity = "8 ounces" });
+    recipe.Ingredients.Add(new RecipeIngredient { Ingredient = beef, Quantity = "1 pound" });
+
+    dbContext.Recipes.Add(recipe);
+    await dbContext.SaveChangesAsync();
+
+    return dbContext.Recipes
+        .Include(r => r.Ingredients)
+        .ThenInclude(ri => ri.Ingredient)
+        .Select(r => new
```

Comment:
Can this `Select` become typed `Select<Recipes.Public.Recipe>` so we make sure to get the DTO type right?

Reply:
Updated. (this initially was just proof that the db works, and will be handled normally when controllers are added)